uaa/src/main/webapp/WEB-INF/spring/saml-providers.xml

```diff
@@ -35,7 +35,7 @@
                 <security:filter-chain pattern="/saml/metadata/**" filters="metadataDisplayFilter" />
                 <security:filter-chain pattern="/saml/idp/metadata/**" filters="idpMetadataDisplayFilter" />
                 <security:filter-chain pattern="/saml/SSO/**" filters="samlSecurityContextPersistenceFilter,samlWebSSOProcessingFilter" />
-                <security:filter-chain pattern="/saml/SingleLogout/**" filters="samlLogoutProcessingFilter" />
+                <security:filter-chain pattern="/saml/SingleLogout/**" filters="samlSecurityContextPersistenceFilter,samlLogoutProcessingFilter" />
```

Comment:
@fhanik  can you please merge this. Issue occurs in scenarios where more than 1 SAML party is involved and if SLO is triggered at this endpoint the IDP initiated the logout.
If this bean is not set here the session is not visible to the logout procedure and therefore issue #967